CountryPicker/CountryPicker/CountryManager.swift

```diff
@@ -66,6 +90,48 @@ open class CountryManager {
     }
 }
 
+
+// MARK: - Country Filter Methods
+public extension CountryManager {
+    
+    ///  Adds a new filter into `filters` collection with no duplicates
+    ///
+    /// - Parameter filter: New filter to be added
+    
+    func addFilter(_ filter: CountryFilterOptions) {
+        
+        var filterSets = Set(self.filters)
+        
+        if filterSets.contains(filter) {
+            return
+        }
+        
+        filterSets.insert(filter)
+        self.filters.removeAll()
+        self.filters.append(contentsOf: Array(filterSets))
```

Comment:
We can directly `filters.instert(filter)` after changing data type to `Set` [insert doc](https://developer.apple.com/documentation/swift/set/3128848-insert)